src/contract_wrappers/modules/transfer_manager/lock_up_transfer_manager_wrapper.ts

```diff
@@ -0,0 +1,804 @@
+    LogWithDecodedArgs<LockUpTransferManagerAddLockUpToUserEventArgs>[]
+  >;
+  (params: GetRemoveLockUpFromUserLogsAsyncParams): Promise<
+    LogWithDecodedArgs<LockUpTransferManagerRemoveLockUpFromUserEventArgs>[]
+  >;
+  (params: GetModifyLockUpTypeLogsAsyncParams): Promise<
+    LogWithDecodedArgs<LockUpTransferManagerModifyLockUpTypeEventArgs>[]
+  >;
+  (params: GetAddNewLockUpTypeLogsAsyncParams): Promise<
+    LogWithDecodedArgs<LockUpTransferManagerAddNewLockUpTypeEventArgs>[]
+  >;
+  (params: GetRemoveLockUpTypeLogsAsyncParams): Promise<
+    LogWithDecodedArgs<LockUpTransferManagerRemoveLockUpTypeEventArgs>[]
+  >;
+  (params: GetPauseLogsAsyncParams): Promise<LogWithDecodedArgs<LockUpTransferManagerPauseEventArgs>[]>;
+  (params: GetUnpauseLogsAsyncParams): Promise<LogWithDecodedArgs<LockUpTransferManagerUnpauseEventArgs>[]>;
+}
+
+interface LockupsParams extends TxParams {
+  lockupName: string;
+}
+
+interface UserAddressParams {
+  user: string;
+}
+
+interface GetTokensByPartitionParams {
+  partition: Partition;
+  tokenHolder: string;
+  additionalBalance: BigNumber;
+}
+
+interface VerifyTransferParams {
+  from: string;
+  to: string;
+  amount: BigNumber;
+  data: string;
+}
+
+interface LockUpTypeParams extends TxParams {
+  lockupAmount: BigNumber;
+  startTime: Date;
+  lockUpPeriodSeconds: BigNumber;
+  releaseFrequenciesSeconds: BigNumber;
+  lockupName: string;
+}
+
+interface LockUpTypeMultiParams extends TxParams {
+  lockupAmounts: BigNumber[];
+  startTimes: Date[];
+  lockUpPeriodSeconds: BigNumber[];
+  releaseFrequenciesSeconds: BigNumber[];
+  lockupNames: string[];
+}
+
+interface LockUpByNameParams extends TxParams {
+  userAddress: string;
+  lockupName: string;
+}
+
+interface LockUpByNameMultiParams extends TxParams {
+  userAddresses: string[];
+  lockupNames: string[];
+}
+
+interface AddNewLockUpToUserParams extends TxParams {
+  userAddress: string;
+  lockupAmount: BigNumber;
+  startTime: Date;
+  lockUpPeriodSeconds: BigNumber;
+  releaseFrequenciesSeconds: BigNumber;
+  lockupName: string;
+}
+
+interface AddNewLockUpToUserMultiParams extends TxParams {
+  userAddresses: string[];
+  lockupAmounts: BigNumber[];
+  startTimes: Date[];
+  lockUpPeriodSeconds: BigNumber[];
+  releaseFrequenciesSeconds: BigNumber[];
+  lockupNames: string[];
+}
+
+interface RemoveLockUpFromUserParams extends TxParams {
+  userAddress: string;
+  lockupName: string;
+}
+
+interface RemoveLockUpFromUserMultiParams extends TxParams {
+  userAddresses: string[];
+  lockupNames: string[];
+}
+
+interface RemoveLockUpTypeParams extends TxParams {
+  lockupName: string;
+}
+
+interface RemoveLockUpTypeMultiParams extends TxParams {
+  lockupNames: string[];
+}
+
+// // Return types ////
+interface LockUp {
+  lockupAmount: BigNumber;
+  startTime: Date;
+  lockUpPeriodSeconds: BigNumber;
+  releaseFrequencySeconds: BigNumber;
+}
+
+interface GetLockUp extends LockUp {
```

Comment:
This should be called `LockUpWithAmount`. I know, horrible, but we should avoid naming interfaces based on which function returns them